pkgs/aws/amazon-genomics-cli/registry.yaml

```diff
@@ -0,0 +1,20 @@
+packages:
+  - type: github_release
+    repo_owner: aws
+    repo_name: amazon-genomics-cli
+    description: Amazon Genomics CLI is an open source tool for genomics and life science customers that simplifies and automates the deployment of cloud infrastructure
+    supported_envs:
+      - amd64
```

Comment:
```suggestion
      - amd64
      - darwin
```

darwin/arm64 is supported by Rosetta2.
In `asset`, `{{.Arch}}` isn't included so we don't have to specify `rosetta2: true`.